src/reducers/app/reducers.ts

```diff
@@ -1,16 +1,54 @@
+import { Token, UserInfo, AppState, UserState } from "../../types";
+import { Actions, AccessTokenTypeKeys, UserInfoTypeKeys } from "./types";
+import { ActionTypeStates } from '../../models'
 
-
-
-export interface AppState {
-
-}
-
-let initialState: AppState = {
+let initialTokenState: Token = {
+  accessToken: null,
 };
+/**
+ * token
+ * @param state
+ * @param action
+ */
+export const token = (
+  state: Token = initialTokenState,
+  action: Actions
+): Token => {
+  switch (action.type) {
+    case AccessTokenTypeKeys.SET_ACCESS_TOKEN:
+      state = { ...state, ...action.payload };
+      break;
+    case AccessTokenTypeKeys.REMOVE_ACCESS_TOKEN:
+      state = { ...state, ...initialTokenState };
+      break;
+    default:
+      break;
+  }
+  return state;
+};
+/**
+ * user
+ */
+let initialUserInfoState: UserState = {
+  status: ActionTypeStates.INPROGRESS,
+	redirectTo: null,
+	error: null,
+  userInfo: null
 
-export const app = (state = initialState, action) => {
+};
+export const userInfo = (
```

Comment:
same ^

Reply:
I cannot find the code snippet in /src/reducers/app/reducers.ts.
Can you please check again?